config/schemas/6/rapidast_schema.json

```diff
@@ -0,0 +1,605 @@
+                                        "type": {
+                                            "type": "string",
+                                            "default": "none",
+                                            "enum": [
+                                                "podman",
+                                                "none"
+                                            ]
+                                        },
+                                        "parameters": {
+                                            "type": "object",
+                                            "properties": {
+                                                "validReturns": {
+                                                    "type": "array",
+                                                    "items": {
+                                                        "type": "integer"
+                                                    }
+                                                },
+                                                "image": {
+                                                    "type": "string"
+                                                },
+                                                "command": {
+                                                    "type": "string"
+                                                },
+                                                "podName": {
+                                                    "type": "string"
+                                                },
+                                                "volumes": {
+                                                    "type": "array",
+                                                    "items": {
+                                                        "type": "integer"
+                                                    }
+                                                }
+                                            },
+                                            "required": []
+                                        }
+                                    }
+                                },
+                                {
+                                    "$comment": "An empty 'container' block will be interpreted as type 'none'",
+                                    "type": "null"
+                                }
+                            ]
+                        },
+                        "defectDojoExport": {
+                            "type": "object",
+                            "properties": {
+                                "parameters": {
+                                    "type": "object",
+                                    "$comment": "This options will directly be sent as parameters to DefectDojo's import/reimport endpoints"
+                                }
+                            }
+                        }
+                    }
+                },
+                {
```

Comment:
According to the [config-template-nessus](https://github.com/RedHatProductSecurity/rapidast/blob/validate-config-jsonschema/config/config-template-nessus.yaml#L17) config file, the `general` block can be empty, which would not validate against the `object` type, so adding type `null`

Reply:
yes the `general` block can be empty.